docs/source/ecosystem/flash.rst

```diff
@@ -0,0 +1,76 @@
+Lightning Flash
+===============
+
+`Lightning Flash <https://lightning-flash.readthedocs.io/en/latest/>`_ is a high-level deep learning framework for fast prototyping, baselining, finetuning and solving deep learning problems.
+Flash makes complex AI recipes for over 15 tasks across 7 data domains accessible to all.
+It is built for beginners with a simple API that requires very little deep learning background, and for data scientists, Kagglers, applied ML practitioners and deep learning researchers that
+want a quick way to get a deep learning baseline with advanced features PyTorch Lightning offers.
+
+.. code-block:: bash
+
+    pip install lightning-flash
+
+-----------------
+
+*********************************
+Using Lightning-Flash in 3 Steps!
+*********************************
+
+1. Load your data
+-----------------
+
+All data loading in Flash is performed via a `from_*` classmethod on a `DataModule`.
+Which `DataModule` to use and which `from_*` methods are available depends on the task you want to perform.
+For example, for image segmentation where your data is stored in folders, you would use the [`from_folders` method of the `SemanticSegmentationData` class](https://lightning-flash.readthedocs.io/en/latest/reference/semantic_segmentation.html#from-folders):
+
+.. code-block:: python
+
+    from flash.image import SemanticSegmentationData
```

Comment:
wondering if we shall rather showcase some more stable like image classification as most of the IceVision integrations are now under reconstruction...
@ethanwharris

Reply:
Semantic segmentation should be fine, that one doesn't use IceVision :smiley: